etc/libconfig.toml

```diff
@@ -28,7 +28,7 @@
 [vpc]
   vpc_enabled = true
   provider_type = "gc"
-  vpc_block_provider_name = "vpc-classic"
+  vpc_provider_type = "vpc-block"
```

Comment:
we need to think in terms of as follows 

vpc is two type classic(gc) and another is gen2(gc2)

so vpc_provider_name=gc/gc2
vpc_volume_type=vpc-file/vpc-block

also try to see what all properties set in the storage-secret-store in the cluster for gen2 and accordingly try to optimise it

Reply:
there is already provider_type to address "gc/gc2" and vpc_block_provider_name as per your suggestion I am not changing. So I am going to introduce vpc_volume_type and let you know the implications. Storage-secret-store has no more things to segregate. But the only redundant property is "vpc_block_provider_name" which is of no use. We can go ahead and remove it later as you suggest.